students/drovdahl/codebat/warmup-2/array_count9.py

```diff
@@ -0,0 +1,9 @@
+#! /usr/local/bin/python3
+
+
+def array_count9(nums):
```

Comment:
There's also a .count() method! Maybe it's cheating, but this is another option:
def array_count9(nums):
    return nums.count(9)